draft-kazuho-httpbis-priority.md

```diff
@@ -29,9 +29,15 @@ informative:
 
 --- abstract
 
-This document describes the Priority HTTP header field.  This header field can
-be used by endpoints to specify the absolute precedence of an HTTP response
-in an HTTP-version-independent way.
+This document describes a prioritization scheme for the HTTP protocol that is
+independent to the HTTP version being used.  It defines a header field used to
+communicate the initial priority, as well as HTTP/2 and HTTP/3 frames for
+reprioritizing the responses.
+
+The priority is expressed using absolute values for each HTTP message, rather
+than expressing the relative precedence between a group of HTTP messages.  The
+structure of the Priority header field and the frames is designed to provide
+future extensibility.
```

Comment:
I like this text but I might change the order of words a little bit. Just my 2c, perhaps too closed-ended :

"This document describes a version-independent prioritization scheme for the HTTP protocol that applies to the resource allocation of HTTP response message payload body. The scheme expresses absolute values for each HTTP message, rather than expressing the relative precedence between a group of HTTP messages. 

This document defines the Priority header field used to communicate the initial priority, as well as HTTP/2 and HTTP/3 frames for reprioritization. These share a common format structure that is designed to provide future extensibility."

Reply:
I was purposefully vague on whether the response resource allocation affects server or client, because we might want to inform client behaviour when it comes to reading.

Reply:
Thank you for the suggestion. I like the shuffling, and updated the abstract in commits up to 5fae1f1.

* I've gone with "delivery" rather than "resource allocation", as the latter seemed a bit confusing.
* s/HTTP response message payload body/HTTP response/. I do not think we (need to) specify if the prioritization scheme is specific to the body (but not to the headers).
* Moved "version independent" to the description of the header field. Frames are definitely version-dependent.